mimesis/providers/internet.py

```diff
@@ -89,7 +89,7 @@ def emoji(self):
         return self.random.choice(EMOJI)
 
     @staticmethod
-    def image_placeholder(width='400', height='300'):
+    def image_placeholder(width: str='400', height: str='300') -> str:
```

Comment:
We should support both: `str` and `int`. I suggest new type `Size = Union[str, int]`